### PR TITLE
Added precision setting in locus methods

### LINKED ISSUE #319: Imaging Locus methods do not set the achieved precision
These values should be set based on the precision achieved by distortion calculations for the framer and LS. I'm not 100% sure what they should be for the SAR. This is somewhat related to #244 because we need to get the final precision out of the distortion model for this. Before that is fixed we should just set it to 0.

### PR DESCRIPTION
**Pull request by jessemapel**
Also finished implementing the framer's proximate imaging locus method.

Fixes #319

### CODE REVIEW

File: tests/FrameCameraTests.cpp [hunk 1]
```diff
@@ -130,6 +130,52 @@ TEST_F(OrbitalFrameSensorModel, GroundPartials) {
...
+  lookY /= lookMag;
+  lookZ /= lookMag;
+  EXPECT_NEAR(locus.direction.x, lookX, 1e-10);
+  EXPECT_NEAR(locus.direction.y, lookY, 1e-10);
+  EXPECT_NEAR(locus.direction.z, lookZ, 1e-10);
+  EXPECT_NEAR(locus.point.x, 1050000.0, 1e-10);
+  EXPECT_NEAR(locus.point.y, 0.0, 1e-10);
+  EXPECT_NEAR(locus.point.z, 0.0, 1e-10);
+  EXPECT_LT(precision, 0.001);
+  EXPECT_TRUE(warnings.empty());
```
Comment (jessemapel) [inline tests/FrameCameraTests.cpp:154]: Basically taken from the linescan test

**Comment by jessemapel** [comment]
Running this through clang tidy seems to have also done some re-ordering of the includes and modified a few other things too. @kberryUSGS  Is this expected?

**Comment by kberryUSGS** [comment]
It will re-order the includes, at least.

**Comment by jessemapel** [comment]
@kberryUSGS Am I good to merge this or should I wait until the other PRs come in because this may cause issues with linting.

**Comment by kberryUSGS** [comment]
@jessemapel if you're willing to wait (and possibly re-run the linted after merging from upstream), it would probably make the merge conflicts easier to handle.

**Comment by jessemapel** [comment]
Closing. Going to open up a clean PR without linting in it.

**Pull request closed without merging**